evasquez26/angular4-startup
# S6-ONE-209 Delete classes from courses

## Story / Task

[ONE-209](https://bixlabs.atlassian.net/browse/ONE-209)

## What this PR does?

Allows a teacher to delete classes from a course he owns.


### Before 

Teacher could not delete classes from courses.

### After

Teacher owner of a course can delete classes from their courses.

### How to test / reproduce

- Enter edition of course you own
- Navigate to classes tab
- Create a class or select a previously created one
- Click the options button
- Click delete
- Clicking Ok on the confirmation modal should delete the class from the course
- Clicking cancel should not delete the course from the list

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thank you!
